service/app/controllers/usermanagement/validator/ShadowUserMigrateReqValidator.java

```diff
@@ -12,11 +12,17 @@
 import java.util.Arrays;
 import java.util.List;
 
+
+/**
+ * @author anmolgupta
+ * this class will be responsilbe to validate the Shadow user migration request.
+ */
 public class ShadowUserMigrateReqValidator extends BaseRequestValidator {
 
     private Request request;
     private String callerId;
-    private static List<String> mandatoryParamsList= new ArrayList<>(Arrays.asList(JsonKey.USER_ID,JsonKey.USER_EXT_ID));
+    private static final List<String> allowedActions=new ArrayList<>(Arrays.asList("accept","reject"));
```

Comment:
use userAction enum for accept and reject

Reply:
done